### ISSUE TITLE
[help wanted] Example doesn't build

### ISSUE THREAD

**Issue by Slethael** [intuit/CardParts#120]
Hi there,

First of all, I'd like to point out that I'm still a noob in Swift development and I struggled installing this Pod but existant issues here helped me until now.

I try to make the first example in the readme work in a very simple workspace, but the two lines above give me an error : 
```
viewModel.title.asObservable().bind(to: titlePart.reactive.title).disposed(by: bag)
viewModel.text.asObservable().bind(to: textPart.reactive.text).disposed(by: bag)
```
The errors: 
```
Value of type 'CardPartTextView' has no member 'reactive'.
Value of type 'CardPartTitleView' has no member 'reactive'.
```

Xcode : 10.1 (10B61)
Swift: 4.2
Target deployment: iOS 11

Is there anyone having the same issue ?

Thanks in advance for your help :)

**Comment by thunpisit** [comment]
I am currently have the same issues https://github.com/intuit/CardParts/issues/117

**Comment by croossin** [comment]
@Slethael @thunpisit We will take a look at these right now. Stand by

**Comment by croossin** [comment]
@Slethael Thanks to @thunpisit we found that we had a little type in our README! Instead of `.reactive.` it should have been `.rx.`.  We have gone ahead and updated the README and it should be correct now.  Thanks again.

**Comment by croossin** [comment]
Fixed by version [2.7.5](https://github.com/intuit/CardParts/releases/tag/2.7.5).
